frontend/src/utils/documentationLinks.js

```diff
@@ -20,7 +23,6 @@ const sampleCode = `${operatorsFramework}/operator-sdk-samples/blob/master/memca
 const capabilityLevelModelDiagram = `${operatorSdk}/blob/master/doc/images/operator-maturity-model.png`;
 const manageOperatorWithOlm = `${operatorsFramework}/getting-started#manage-the-operator-using-the-operator-lifecycle-manager`;
 const operatorGroupDesign = `${olm}/blob/master/Documentation/design/architecture.md#operator-group-design`;
-const addingYourOperator = `${operatorsRepo}#adding-your-operator`;
```

Comment:
This causes the build to fail because the `const` is referenced further below.

```
ERROR in ./src/utils/documentationLinks.js 63:331
--
  | Module parse failed: Export 'addingYourOperator' is not defined (63:331)
```